Description: Fix tooltip_trigger_hide function declaration
 tooltip_trigger_hide does not have a parameter in tooltip.h,
 so the code bears the risk of undefined behaviour.
 .
 This patch simply removes the argument, since the function
 does not use the parameter at all.
Author: Sebastian Reichel <sre@debian.org>
Bug-Debian: 748171
Last-Update: 2014-06-01

Index: tint2/src/tooltip/tooltip.c
===================================================================
--- tint2.orig/src/tooltip/tooltip.c
+++ tint2/src/tooltip/tooltip.c
@@ -259,7 +259,7 @@ void tooltip_update()
 }
 
 
-void tooltip_trigger_hide(Tooltip* tooltip)
+void tooltip_trigger_hide()
 {
 	if (g_tooltip.mapped) {
 		tooltip_copy_text(0);
